bundles/org.openhab.binding.astro/src/main/resources/OH-INF/config/config.xml

```diff
@@ -53,6 +53,11 @@
 			<description>The latest time of the day for the event or the datetime value (hh:mm).</description>
 			<context>time</context>
 		</parameter>
+		<parameter name="forceEvent" type="boolean" required="true">
```

Comment:
No need to mark the parameter as required as you provide a default value.

Reply:
ah, thanks thought it would was necessary to also list it as required.